album/pokemon-scarlet-violet-super-music-collection.yaml

```diff
@@ -0,0 +1,1900 @@
+Album: 'Pokémon Scarlet & Pokémon Violet + The Hidden Treasure of Area Zero: Super Music Collection'
+Directory: pokemon-scarlet-violet-super-music-collection
+Artists:
+- Pokémon
+- GAME FREAK
+Date: February 27, 2024 00:00:02
+## https://vgmdb.net/album/134816
+Date Added: 
+URLs:
+- https://www.youtube.com/playlist?list=PLjjzf1NYDNrlrtLn41YRK1uPjQZl8GnvX
+Cover Artists:
+- Toshiyuki Takahashi
+- Hiromi Yasuda
+- Momoko Fukuda
+- 上野愛恵 ##can't find who this is lol
+Cover Art File Extension: png
+Color: '#f8ce00'
+Groups:
+- Toby Fox
+- Beyond
+Wallpaper Artists:
+- GAME FREAK
```

Comment:
Is there a reason this is listed as GAME FREAK instead of Pokémon? Same goes for SwSh

Reply:
the art is made by someone from GAME FREAK, or a third party artist that was commisioned to make the art. since we don't know what individual made the art, I put GAME FREAK to reflect that the art was "made" by GAME FREAK, without any further details available.